jekyll/_cci2/authentication/mfa.adoc

```diff
@@ -0,0 +1,71 @@
+With MFA enabled, an additional verification step is required to access your account and make changes to account authentication settings, such as email or password.
+This means that even if your account password is compromised, only someone with access to the additional verification factor can
+access the account. The additional verification step takes the form of providing a One-Time Password (OTP).
+
+[#setup-mfa]
+== Setup MFA
+
+To configure MFA on your account, follow these steps:
+
+. In the link:https://app.circleci.com/home/[CircleCI web app], select your profile from the upper right corner, then select **User Settings**.
+. Select **Password** from the sidebar.
+. On the Password & authentication page, in the **Multi-factor authentication** section, select **Add authenticator app**. MFA is marked as **Not enabled** until a factor is added.
+. Input your password at the password prompt.
+. At the **Add authenticator app** prompt, scan the provided QR code using an authenticator app or browser extension. Then verify the code generated by the app, by inputting it into the provided text box.
+. At the next screen, you are provided with the MFA recovery code. You must copy this code and save it somewhere safe. This code will only be displayed once and is the last resort for accessing your account in the event that you lose access to your MFA factor. Without this code, you would lose access to your account.
+. You may close the window to complete the setup. When MFA is successfully configured, it is marked as **Enabled** on the Password & authentication page.
+
+image::../../img/docs/authentication/mfa-enabled.png[Password and authentication page with MFA enabled]
+
+[#using-mfa]
+== Using MFA on your account
+
+Once you have MFA enabled, then you must provide a second verification factor whenever you log into your account.
+
+image::../../img/docs/authentication/mfa-otp-login.png[Login page with request for OTP]
+
+You will also be asked to provide a second factor whenever you attempt to change the authentication settings of your account (for example updating your email or password).
+
+Currently CircleCI supports MFA using authenticator applications only. Once you have configured MFA, as per the previous section, then you must use the OTP generated by
+the application whenever this is requested in the CircleCI web app. The OTP will be displayed in the app, and you must input this code when prompted in the CircleCI web app.
+
+You may disable MFA in the **Password and authentication** page by deleting your OTP factor. If you do this, then we recommend that you re-enable MFA at your earliest convenience for the security benefit it provides.
+
+[#mfa-recovery-codes]
+== MFA recovery codes
+
+WARNING: If you have MFA enabled, and lose access to your MFA factor and recovery code, then you will lose access to your account. You must ensure that you do not lose the recovery code.
```

Comment:
~~Should we allow support to be able to do recoveries? It would require identifying that the user is who they say they are